os-partition/added/partitionPV.sh

```diff
@@ -48,6 +48,11 @@ function partitionPV() {
           SERVER_DATA_DIR="${INSTANCE_DIR}/serverData"
           mkdir -p "${SERVER_DATA_DIR}"
 
+          if [ -f /opt/partition/copyDataDir.sh ]; then
+            source /opt/partition/copyDataDir.sh
+            copy_data_dir ${SERVER_DATA_DIR}
+          fi
+ 
```

Comment:
This should be a function call just like runServer, defined in the script sourcing the partitionPV.sh.

Reply:
Once the function is defined in the sourcing script this could be

init_data_dir "${SERVER_DATA_DIR}"